lib/iris/fileformats/pp.py

```diff
@@ -673,8 +673,8 @@ def __init__(self, value):
 
         """
         if value < 0:
-            raise ValueError('Negative numbers not supported with '
-                             'splittable integers object')
+            msg = 'Setting LBPROC with a negative value of {}'
```

Comment:
Or if we really to have to allow such things, just ditch the warning? There's nothing the user can do to avoid it...